pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "strawberry-graphql"
-packages = [ { include = "strawberry" } ]
+packages = [{ include = "strawberry" }]
```

Comment:
I discourage formatting changes mixed with other changes. Particularly for a PR already of this magnitude. It makes it so much more difficult to see what's actually changed.

Reply:
Oh, totally! I'm also the one who comments that on PRs I review. My autoformatter formatted this when I modified this file and I didn't even notice =P